gensim/test/test_ldamodel.py

```diff
@@ -249,6 +249,26 @@ def testGetDocumentTopics(self):
                 self.assertTrue(isinstance(k, int))
                 self.assertTrue(isinstance(v, float))
 
+        #Test case to use the get_document_topic function for the corpus
```

Comment:
Please add a new test for `minimum_prob` and `minimum_phi_value`. Select values that change the number of topics and docs returned